doc/tutorials/mount.md

```diff
@@ -1,56 +1,146 @@
-Elektra provides a global key database where configuration of all
-applications can be integrated.
+# Mounting #
 
-There are different forms of integration.  In this tutorial we will
-discuss a light form of integration that can be achieved without
-elektrifying applications.
+Elektra provides a global key database where configuration of all applications can be integrated.
 
-The idea is to manipulate the configuration values directly in the
-configuration files via the key database.  Thus the applications will
-read the configuration file, changes in the key database will be picked
-up by applications.
+There are different forms of integration. In this tutorial we will discuss a light form of integration that can be achieved without [elektrifying](/doc/help/elektra-glossary.md) applications.
 
-Goals of this Tutorial:
+The idea is to synchronize Elektras configuration with configuration files. Thus applications can read the configuration files and changes in the key database will be picked up by applications.
 
-- [ ] Understanding Backends
-- [ ] Understanding Mounting
-- [ ] Understanding Plugins
-- [ ] Understanding get/set
+The heart of the approach is the so called _mounting_ of configuration files into the key database.
 
-## Limitations
+Let us start with a motivating example first:
 
-The approach is not ideal when the application itself also changes the
-configuration thus Elektra is bypassed then.  But th Furthermore one
-cannot easily change the configuration format file format (it must be
-in the format the application understands) thus one loses quite some
-flexibility.
+## Mount the lookup table for hostnames
 
-But many applications are only reading the configuration file and for
-many setups it is acceptable not being able to change the file format.
-The heart of the approach is the so called mounting of configuration files
-into the key database.  Note that mounting requires root permissions.
+We mount the lookup table with the following command:
 
     sudo kdb mount /etc/hosts system/hosts hosts
 
-1. The first argument is the configuration file we want to mount, 2. the
-second which path it should have in the key database, and 3. the third
-is the plugin that can read and write this configuration format.
+1. `/etc/hosts` is the configuration file we want to mount
+2. `system/hosts` is the path it should have in the key database, also known as **mountpoint**
+3. `hosts` is the _storage plugin_ that can read and write this configuration format.
 
-We can verify that the file is mounted via:
+Without arguments `kdb mount` lists the currently mounted backends.
 
-    kdb file system/hosts
+    $ kdb mount
+    /etc/hosts on system/hosts with name system/hosts
 
-After mounting a file, we can modify the keys below `system/hosts`.
-We need again to be root, because we modify `/etc/hosts`.
+Which verifies, that all configuration below `system/hosts` is stored in /etc/hosts.
```

Comment:
This is unfortunately wrong for all namespaces except `system` and `spec` (@sanssecours also fell into the trap). Between the path displayed in `kdb mount` there is an chroot-like abstraction. For different namespaces, the path is different. Verification, of where a file is, is possible via `kdb file`, not via `kdb mount` (it only verifies what is currently mounted).

See `kdb info resolver` for more info about this chroot-like abstraction.

Please also add this information in the tutorial.